### ISSUE TITLE
Embed/transclusion has partial left margin

### ISSUE THREAD

**Issue by DutchPete** [nikbrunner/obsidian-deep-work-theme#18]
**Describe the bug**
An embed/transclusion in Preview shows a partial sidebar in accent color. It should not be there.
The screenshots explain what I mean.

**To Reproduce**

**Expected behavior**

**Screenshots**
Embed in Edit (Live Preview)
![image](https://user-images.githubusercontent.com/10794265/144799247-0d485105-7b3b-4237-b80e-42f2d11af2ef.png)

Same embed in Preview:
![image](https://user-images.githubusercontent.com/10794265/144799322-f52f2536-0a3c-4814-8623-025269070d1a.png)

The blue bit on the left should not be there.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] MacOS 10.14.6

**Smartphone (please complete the following information):**

**Additional context**

**Comment by nikbrunner** [comment]
I will check on it tonight. Thank you for reporting

**Comment by DutchPete** [comment]
Thanks for your time and a nice theme: simple and attractive without a plethora of colors.

**Comment by nikbrunner** [comment]
I couldn't reproduce this until now. Is it possible that the your file which you want to embed is empty?

**Comment by DutchPete** [comment]
Hi Nik, this is the part in your theme that causes the issue:
```css
body
  .markdown-preview-view
  .markdown-embed
  .markdown-preview-view {
  padding-left: 1.5rem;
  padding-right: 1.5rem;
  border-left: 10px solid var(--text-accent);
  border-radius: 4px;
  background: var(--background-secondary);
}
```

And this is the snippet I have written to counter the issue:
```css
body
.markdown-preview-view
.markdown-embed
.markdown-preview-view {
    border-left: none;
}
```

**Comment by nikbrunner** [comment]
I will check on it tonight, thank you very much :)

**Comment by DutchPete** [comment]
Don't feel you have to change this for me. If you cannot reproduce it and/or no one else has commented about it, then you may as well leave it, I don't mind using the snippet.

**Comment by AToMiXhawK** [comment]
Similar issue with code blocks

In Edit mode:

![image](https://user-images.githubusercontent.com/42279685/145135294-7b27c2a0-c27f-4e55-ab4a-d8d36c9e83ad.png)

In Preview mode:

![image](https://user-images.githubusercontent.com/42279685/145135327-7d93e5d7-7627-45ac-bc8d-d104a0f46ffa.png)


The first line is a little off to the right, which is annoying me.

**Comment by DutchPete** [comment]
I don't use code in Obsidian in a note (I don't consider Admonitions to be real code) so I don't have that problem. As a non-CSS expert I think your code block issue is different from what I described above; I managed to solve that one with CSS.

Yesterday there was a discussion on the `#appearance` channel about the code block issue you have. I think they found the problem and a fix.

**Comment by nikbrunner** [comment]
> Similar issue with code blocks
> 
> In Edit mode:
> 
> ![image](https://user-images.githubusercontent.com/42279685/145135294-7b27c2a0-c27f-4e55-ab4a-d8d36c9e83ad.png)
> 
> In Preview mode:
> 
> ![image](https://user-images.githubusercontent.com/42279685/145135327-7d93e5d7-7627-45ac-bc8d-d104a0f46ffa.png)
> 
> The first line is a little off to the right, which is annoying me.

This should be fixed with this: dadcd957c2153e3bbfbcf668b98e63d0d98cc699. 
You can try to update the theme and see if the problem persists.

But I will test both scenarios tonight after work and try to replicate these behaviours and commit a fix.
Thanks guys

**Comment by nikbrunner** [comment]
Along with Styling Improvements for Checkboxes and Headings, I updated the styles for embedded content and fixed some problems with it. I think its better now. 

I did not remove the colored border to the left for embedded stuff. I want to keep it as a styling choice and not completely remove it. After the fixes, I could not find any more errors on my end with it.

@DutchPete If there still exists some problem with it, I would need some example markdown code and the used mode which reproduces the problem. Also keep in mind that other factor like snippets etc can have unexpected side effects.

I will close this issue for now. Please reopen it, if you need help :)
